src/m365/spo/commands/contenttype/contenttype-get.ts

```diff
@@ -53,13 +53,22 @@ class SpoContentTypeGetCommand extends SpoCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        if (res['odata.null'] === true) {
-          if(args.options.id){
-            cb(new CommandError(`Content type with ID ${args.options.id} not found`));
-          }
-          if(args.options.contenttypeTitle){
-            cb(new CommandError(`Content type with title ${args.options.contenttypeTitle} not found`));
-          }
+
+        let errorMessage: string = '';
+
+        if (args.options.name) {
+          res = res.value;
```

Comment:
This will return an array rather than the content type object, whereas retrieving the content type by ID returns the content type object. To keep both variants in sync we'd need to use:

```suggestion
          res = res.value[0];
```

Reply:
thanks for the tip 👍 .. done 😊